src/validators/credit.js

```diff
@@ -18,27 +18,34 @@ const creditCounselingModel = {
   },
 }
 
+export const validateFinancialCredit = data => (
+  validateModel(data, creditCounselingModel) === true
+)
+
 export default class CreditValidator {
   constructor(data = {}) {
     this.data = data
   }
 
   validHasCreditCounseling() {
-    return validateModel(this.data, { HasCreditCounseling: creditCounselingModel.HasCreditCounseling }) === true
+    return validateModel(this.data, {
+      HasCreditCounseling: creditCounselingModel.HasCreditCounseling,
+    }) === true
```

Comment:
really bummed that github isn't catching these as whitespace changes. I guess it's the trailing comma.